generator/tree.go

```diff
@@ -214,6 +214,9 @@ func generateConfigModule(cfg *ModuleConfig, node *Node, nameToNode map[string]*
 					if !ok {
 						log.Fatalf("Unknown index type %s for %s", indexNode.Type, lookup.NewIndex)
 					}
+					if cfg.Overrides[sanitizeLabelName(indexNode.Label)].Type != "" {
```

Comment:
This is not the only place types are used, you need to produce a copy of the tree with this changed.